Dockerfile

```diff
@@ -0,0 +1,52 @@
+# hadolint global ignore=DL3008
+FROM debian:buster
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+      python \
+      python-pip \
+      python-dev \
+      python-setuptools \
+      ca-certificates \
+      gcc \
+      libpq-dev \
+      curl \
+      gunicorn \
+    && rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# kelseyhightower confd arm64 builds are broken: https://github.com/kelseyhightower/confd/issues/850
+ARG CONFD_VERSION=0.19.1
+RUN CONFD_ARCH=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) \
+    && curl -SL "https://github.com/abtreece/confd/releases/download/v${CONFD_VERSION}/confd-v${CONFD_VERSION}-linux-${CONFD_ARCH}.tar.gz" | tar -xz -C /usr/local/bin/ \
+    && /usr/local/bin/confd -version
+
+RUN groupadd --system --gid 510 osqa && useradd --system --gid osqa --uid 510 osqa
+
+WORKDIR /usr/src/app
+RUN chown osqa:osqa /usr/src/app
+
+COPY requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY --chown=osqa . .
+
+# Add confd templates
+COPY docker/confd/ /etc/confd/
+
+# Set Defaults
+ENV OSQA_DB_ENGINE=django.db.backends.postgresql_psycopg2
+ENV OSQA_DB_NAME=osqa
+ENV OSQA_DB_USER=osqa
+ENV OSQA_DB_PASSWORD=osqa
```

Comment:
If you're going to use network connections and password authentication for the database then this needs to be a secure password and not be exposed in a public repository.

Reply:
These are only the "default" credentials and will not use them in production.
We will not use the `docker-compose.yml` file in production